packages/teleport/src/services/nodes/makeNodeBashCmd.ts

```diff
@@ -14,13 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import moment from 'moment';
+import { formatDistanceStrict } from 'date-fns';
 import cfg from 'teleport/config';
 import { NodeToken, BashCommand } from './types';
 
 export default function makeNodeBashCmd(token: NodeToken): BashCommand {
-  const duration = moment(new Date()).diff(token.expiry);
-  const expires = moment.duration(duration).humanize();
+  const expires = formatDistanceStrict(new Date(), new Date(token.expiry));
```

Comment:
confused on this one, the typing implies that `token.expiry` should already come as date format, but we are formatting it again. Isn't `token: Nodetoken` a  json response? If it is, i'd remove the typing and rename it to `json`